### PR TITLE
Add update author test

### PR DESCRIPTION
**Pull request by jmmxp**
Test uses PUT as is, may need to be changed to POST in the future. Should be as easy as changing `.put` to `.post`.

### CODE REVIEW

File: back-end/api/quickstart/tests.py [hunk 1]
```diff
@@ -31,6 +32,36 @@ def test_get_invalid_author(self):
...
+
+  # TODO: update object tests most likely will be changed to use POST instead.
+  def test_update_author(self):
+    response = client.put(
+      f'/authors/{self.john._id}/',
+      data=json.dumps(self.payload),
+      content_type='application/json'
+    )
+    self.assertEqual(response.status_code, status.HTTP_200_OK)
+
```
Comment (VinnyLuu) [inline back-end/api/quickstart/tests.py:63]: Should you get the author here and assert if its actually updated?

Comment (jmmxp) [reply back-end/api/quickstart/tests.py:63]: Done.

Comment (jmmxp) [reply back-end/api/quickstart/tests.py:63]: Now checking that the PUT'ed new fields are updated, and the rest are unchanged.

**Review by VinnyLuu — APPROVED** [review]
LGTM

**Pull request merged**